.travis.yml

```diff
@@ -9,6 +9,7 @@ before_install:
   - wget http://www.cmake.org/files/v3.1/cmake-3.1.3-Linux-x86_64.sh
   - mkdir $HOME/cmake
   - sh cmake-3.1.3-Linux-x86_64.sh --skip-license --prefix=$HOME/cmake
+  - npm install jshint
```

Comment:
You should add the line 

    [ -e /usr/bin/node ] || ln -s /usr/bin/nodejs /usr/bin/node

See http://stackoverflow.com/questions/20886217/browserify-error-usr-bin-env-node-no-such-file-or-directory.  This does affect jshint and travis uses ubuntu which is one of the distributions that is affected.